### PR TITLE
Added thoth s2i provenance check

### PR DESCRIPTION
**Pull request by saisankargochhayat**
Demonstrate Thoth s2i provenance check.
Related - #10  
Link to the tutorial - [link](https://www.katacoda.com/saisankargochhayat/scenarios/thoth-provenance)

### CODE REVIEW

File: ai-machine-learning/thoth-s2i-provenance/app.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Before we explore more about the build process, to explore more about the s2i process.
```
Comment (fridex) [inline ai-machine-learning/thoth-s2i-provenance/app.md]: This sentence is not clear to me.

File: ai-machine-learning/thoth-s2i-provenance/app.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          referencePolicy:
+            type: "Source"
+```
+And we use this under BuildConfig under line 26, stating our source repo to be - 
+```
+git:
+    uri: "https://github.com/thoth-station/s2i-example"
+    ref: "log-thoth-broken"
+```
+Now that we have successfully exlpored the repo, and are using a thoth powered s2i image, let's explore some configuration options, that
```
Comment (fridex) [inline ai-machine-learning/thoth-s2i-provenance/app.md]: s/exlpored/explored/

File: ai-machine-learning/thoth-s2i-provenance/parameter.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`THOTH_ERROR_FALLBACK` - fallback to the Pipfile.lock present in the repository if the submitted Thoth analysis fails
+
+If you go checkout the `openshift.yaml` in [s2i-example/log-thoth-broken](https://github.com/thoth-station/s2i-example/blob/log-thoth-broken/openshift.yaml) and checkout the env block under BuildConfig (ln 57), you will see some of these being used like `THOTH_PROVENANCE_CHECK`, `THAMOS_ADVISE`, `THOTH_DRY_RUN`, `THOTH_HOST`.
+Thamos Host is currently set to `api.moc.thoth-station.ninja`, which is our public facing api. 
+And `THOTH_PROVENANCE_CHECK` set to 1, that triggers the provenance check. For this tutorial we have turned of `THOTH_ADVISE=0` to focus on provenance checks only.
+
+
+Here are some config option's that you could configure, which 
+changes the behaviour of `THAMOS` (the cli tool used to interact with Thoth internally.)
```
Comment (fridex) [inline ai-machine-learning/thoth-s2i-provenance/parameter.md]: s/THAMOS/Thamos/

File: ai-machine-learning/thoth-s2i-provenance/deploy.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+Now that you are familiar with the configurations our s2i image offers, let's try deploying our sample app using Thoth's S2I build process. 
+
+Assuming you have followed the steps from before are logged in as an admin into `myproject` in the openshift cluster lets deploy - 
+
+If you haven't forked the repo and made any changes, you can try deploy our version using -
```
Comment (fridex) [inline ai-machine-learning/thoth-s2i-provenance/deploy.md]: I think we can safely say users follow the tutorial and do not fork the repo (not to confuse them).

**Review by fridex — COMMENTED** [review]
Note capitals - Open*S*hift, *T*thoth, *P*ython

Some sentences end with `-` (dash) - is it intentional?

Also, somewhere are S2I elsewhere I see s2i.

Very nice scenario! :clap:

**Comment by saisankargochhayat** [comment]
@fridex Fixed according to the suggestions. Commit - [c8b8684](https://github.com/AICoE/learn-katacoda/pull/26/commits/c8b8684919fcb617c9f0bb9803caa6f1b51f947a)

File: ai-machine-learning/thoth-s2i-provenance/app.md (hunk 1 above)
Comment (saisankargochhayat) [reply ai-machine-learning/thoth-s2i-provenance/app.md]: Reformatted it :)

File: ai-machine-learning/thoth-s2i-provenance/deploy.md (hunk 4 above)
Comment (saisankargochhayat) [reply ai-machine-learning/thoth-s2i-provenance/deploy.md]: Thanks, changed it

**Review by 4n4nd — APPROVED** [review]
:+1: LGTM Great job!

**Pull request merged**